Add vignette

llrs opened the pull request:
Following the README.Rmd I created this PR to add a vignette (related to #2).
While creating the vignette I found is a lack of functions to use the data retrieved. 
Also having a summary of the results obtained printed on the console when performing a query could come handy.

llrs committed:
Add vignette

sckott commented on vignettes/jaod.Rmd:
````diff
@@ -0,0 +1,99 @@
+)
+```
+
+Vignettes are long form documentation commonly included in packages. Because they are part of the distribution of the package, they need to be as compact as possible. The `html_vignette` output type provides a custom style sheet (and tweaks some options) to ensure that the resulting html is as small as possible. The `html_vignette` format:
````
Can you remove this paragraph and bulleted list?  We usually don't introduce the vignette format in vignettes.

llrs replied:
Oh, sorry I forgot to remove it
```suggestion
```

llrs committed:
Remove description of the vignette

sckott commented:
Looks like you need to merge in changes from master before we can merge here

llrs committed:
Pulling from upstream

Merge branch 'master' of https://github.com/ropensci/jaod

# Conflicts:
#	DESCRIPTION
